### DISCUSSION TITLE
Some cyrillic letters shows not correctly

### DISCUSSION THREAD

**Discussion by nacknime-official** [LunarVim/LunarVim#2400] [Q&A] [answered]
Vanilla NVIM works correctly, but Lunarvim shows (or prints, idk) russian and ukrainian letters not correctly.
Instead of "привет" it types "Ð°рÐ¸Ð²Ðµт".

Video:
https://asciinema.org/a/VHzOfrmNRE8RNJr3Hub6dYI4t

**Comment by LostInTheLogs** [comment, marked as answer]
can't reproduce 
![image](https://github.com/LunarVim/LunarVim/assets/110467150/11f6143c-742c-4305-833f-ea9fc328e458)
